### PR TITLE
Merge my slogan

### PR DESCRIPTION
**Pull request by pamichals**

### CODE REVIEW

File: demo.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
-import pandas as pd
+'import pandas as pd
```
Comment (claesmk) [inline demo.py:1]: It looks like this is the reason unit-test is failing - quote in front of the import

File: test_demo.py [hunk 2]
```diff
@@ -8,10 +8,16 @@ def test_show_aggie_pride(self):
         df = show_aggie_pride()
         self.assertEqual(df.loc[0, 'Text'], 'Aggie Pride')
         self.assertEqual(df.loc[1, 'Text'], 'Worldwide')
+
```
Comment (claesmk) [inline test_demo.py:11]: It doesn't look like this will work because you have two slogans in the same integer location (loc=2)

**Review by claesmk — CHANGES_REQUESTED** [review]
Were you able to run the demo and unit-test on your computer?  It looks like there are errors that would have prevented it from running OK

**Comment by pamichals** [comment]
> Were you able to run the demo and unit-test on your computer? It looks like there are errors that would have prevented it from running OK

Yes, both ran successfully

**Review by claesmk — APPROVED** [review]

**Pull request merged**